ocs_ci/ocs/resources/pod.py

```diff
@@ -242,6 +242,9 @@ def run_io(
         if fio_filename:
             self.io_params['filename'] = fio_filename
         self.io_params['iodepth'] = depth
+        if depth > 1:
+            self.io_params['ioengine'] = 'libaio'
```

Comment:
Those are the default values.
Anything should be changed when depth is <=1?

Reply:
Now I see the default. In ocs_ci/templates/workloads/fio/workload_io_rw.yaml
Thanks
Could not find anything that needs to be changed when iodepth <= 1.
Closing.